admin/tool/urlpreview/db/access.php

```diff
@@ -1,35 +0,0 @@
-<?php
-// This file is part of Moodle - https://moodle.org/
-//
-// Moodle is free software: you can redistribute it and/or modify
-// it under the terms of the GNU General Public License as published by
-// the Free Software Foundation, either version 3 of the License, or
-// (at your option) any later version.
-//
-// Moodle is distributed in the hope that it will be useful,
-// but WITHOUT ANY WARRANTY; without even the implied warranty of
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-// GNU General Public License for more details.
-//
-// You should have received a copy of the GNU General Public License
-// along with Moodle.  If not, see <https://www.gnu.org/licenses/>.
-
-/**
- * defines capabilities for access to urlpreview tool
- *
- * @package     tool_urlpreview
- * @copyright   2023 Hanbin Lee <n10324402@qut.edu.au>
- * @license     https://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
- */
-
- defined('MOODLE_INTERNAL') || die();
-
- $capabilities = [
-  'tool/urlpreview:usetool' => [
```

Comment:
We should end up with a capability around who can use the debugging tool. This would typically be given to site admins but also makes sense to be given to any course editor role too by default